stage2/gcp-terraform/main.tf

```diff
@@ -0,0 +1,64 @@
+terraform {
+  required_providers {
+    kubernetes = {
+      source  = "hashicorp/kubernetes"
+      version = "2.22.0"
+    }
+  }
+}
+
+provider "kubernetes" {
+  config_path    = var.k8s_config
+  config_context = var.k8s_context
+}
+
+provider "helm" {
+  kubernetes {
+    config_path    = var.k8s_config
+    config_context = var.k8s_context
+  }
+}
+
+# module "cert_manager" {
+#   source = "../modules/cert-manager/stage2"
+# }
+
+# module "grafana_tempo" {
+#   source    = "../modules/grafana-tempo/stage2"
+#   namespace = var.dmtr_namespace
+# }
+
+# module "postgresql" {
+#   source = "../modules/postgresql/stage2"
+# }
+
+# Nodeexporter deamonset requires defined resource quota as part of critical pods
+module "o11y" {
+  source        = "../../modules/common/o11y/stage2"
+  namespace     = var.dmtr_namespace
+  storage_class = "standard"
+  depends_on    = [kubernetes_resource_quota.critical_pods_quota]
+}
```

Comment:
Comment this out for now w/ a TODO to revisit since we're using GKE Autopilot currently